spark/v3.3/spark/src/test/java/org/apache/iceberg/ValidationHelpers.java

```diff
@@ -42,7 +42,7 @@ public static List<Long> snapshotIds(Long... ids) {
   }
 
   public static List<String> files(ContentFile<?>... files) {
-    return Arrays.stream(files).map(file -> file.path().toString()).collect(Collectors.toList());
+    return Arrays.stream(files).map(file -> file.location()).collect(Collectors.toList());
```

Comment:
nit: We could use method reference. Feel free to ignore this comment. I know Spark v3.5 uses this style.